dev-libs/device-atlas-api-c/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>bertrand@jacquin.bzh</email>
+		<name>Bertrand Jacquin</name>
+	</maintainer>
+	<longdescription>
```

Comment:
Please add proxy-maint project here as well, unless you have some other developer who wants to commit for you.